test/e2e/plain_provisioner_test.go

```diff
@@ -845,7 +845,11 @@ var _ = Describe("plain provisioner bundleinstance", func() {
 					WithTransform(func(c *metav1.Condition) metav1.ConditionStatus { return c.Status }, Equal(metav1.ConditionFalse)),
 					WithTransform(func(c *metav1.Condition) string { return c.Reason }, Equal(rukpakv1alpha1.ReasonInstallFailed)),
 					WithTransform(func(c *metav1.Condition) string { return c.Message },
-						ContainSubstring(`unable to recognize "": no matches for kind "OperatorGroup" in version "operators.coreos.com/v1"`)),
+						Or(
+							ContainSubstring(`unable to recognize "": no matches for kind "OperatorGroup" in version "operators.coreos.com/v1"`),
```

Comment:
Can you add a comment here explaining the scenario that requires this? I feel like we'll quickly forget why this is necessary, which may lead to cargo-culting or at least losing the thread on a possible solution (e.g. wrap both of these errors in the controller such that our test can look for the wrapper)

Reply:
Yeah definitely. I'll add a TODO to get back to this tomorrow.